sceptre/scipool/README.md

```diff
@@ -31,15 +31,44 @@ Requirements:
 * Run `pre-commit install` to install the git hook.
 
 ## Testing
+
+### Pre-Commit
 As a pre-deployment step we syntatically validate our sceptre and
 cloudformation yaml files with [pre-commit](https://pre-commit.com).
 
 Please install pre-commit, once installed the file validations will
-automatically run on every commit.  Alternatively you can manually
+automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
 Please install pre-commit, once installed the file validations will
 automatically run on every commit.
 
+### Functional Testing
+The process to test the functionality of an AMI and it's integration with our
+Service Catalog products is to first create a test AMI, upload a modified
+product template to S3, and create a new Service Catalog product in the
+scipool dev account to verify manually from https://sc-dev.sageit.org/
+
+The deploy pipelines for both our packer repos and our service catalog library
```

Comment:
Can you link to the "packer repo's"?

Reply:
I've added a list to the end of the section